libs/accounts/email-renderer/src/layouts/fxa/mocks.ts

```diff
@@ -0,0 +1,6 @@
+export const includes = {
```

Comment:
I suppose this and some other new `mocks.ts` files could use a license at the top 🤷‍♀️

Reply:
I think I got them all now.